### PR TITLE
session1 exercises

### PR DESCRIPTION
**Pull request by mangel99**

### CODE REVIEW

File: manuel_angel/session01/exercise02/wordpress-canary-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+apiVersion: extensions/v1beta1
```
Comment (juan131) [inline manuel_angel/session01/exercise02/wordpress-canary-deployment.yaml]: No canaries on exercise2!!!!

File: manuel_angel/session01/exercise02/wordpress-canary-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  selector:
+    matchLabels:
+      app: wordpress
+      version: canary
+  template:
+    metadata: 
+      labels:
+        app: wordpress
+        version: canary
+    spec:
```
Comment (juan131) [inline manuel_angel/session01/exercise02/wordpress-canary-deployment.yaml]: I'm missing the **resources** fields

File: manuel_angel/session01/exercise02/mariadb-slave-deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  selector:
+    matchLabels:
+      app: mariadb
+      component: "slave"
+  template:
+    metadata: 
+      labels:
+        app: mariadb  
+        component: "slave"
+    spec:
```
Comment (juan131) [inline manuel_angel/session01/exercise02/mariadb-slave-deployment.yaml]: I'm missing the **resources** fields

File: manuel_angel/session01/exercise02/mariadb-deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  selector:
+    matchLabels:
+      app: mariadb
+      component: "master"
+  template:
+    metadata: 
+      labels:
+        app: mariadb  
+        component: "master" 
+    spec:
```
Comment (juan131) [inline manuel_angel/session01/exercise02/mariadb-deployment.yaml]: I'm missing the **resources** fields

File: manuel_angel/session01/exercise02/mariadb-slave-deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,83 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: mariadbslave
+  namespace: exercise02
+  labels: 
+    app: mariadb
+    component: "slave"
+spec:
```
Comment (juan131) [inline manuel_angel/session01/exercise02/mariadb-slave-deployment.yaml]: I'm missing the **rollingUpdate** strategy definition

File: manuel_angel/session01/exercise02/wordpress-svc.yaml [hunk 6]
```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: wordpress-svc
+  namespace: exercise02
+spec:
+  type: NodePort
+  sessionAffinity: ClientIP
```
Comment (juan131) [inline manuel_angel/session01/exercise02/wordpress-svc.yaml]: It's not needed when not using canary deployments

File: manuel_angel/session01/exercise02/README.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
+# Exercise II
```
Comment (juan131) [inline manuel_angel/session01/exercise02/README.md]: I'm missing a **commands.sh** file to deploy this solution

File: manuel_angel/session01/exercise02/wordpress-deployment.yaml [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+            value: "yes"
+          - name: MARIADB_HOST
+            valueFrom:
+              configMapKeyRef:
+                name: ex01configmap
+                key: MARIADB_HOST
+          - name: MARIADB_PORT_NUMBER
+            valueFrom:
+              configMapKeyRef:
+                name: ex01configmap
```
Comment (juan131) [inline manuel_angel/session01/exercise02/wordpress-deployment.yaml]: This **configmap** doesn't exist

File: manuel_angel/session01/exercise01/README.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
+# Exercise I
```
[the remark this answers is not in the document]
Comment (mangel99) [reply manuel_angel/session01/exercise01/README.md]: First my apologies, 
I thought you wouldn't dedicate to work on making a deep revision as you did.
I take it as a little exercise to practice, without giving it too much thought, but due to your implication, I'll take your comments to fix it.

Obviously, I didn't do the exercise for a productive or real environment (security, ports...etc). I couldn't dedicate more than 3 hours the Wednesday night to do it and now I feel bad after your effort to correct it with so much detail.

File: manuel_angel/session01/exercise01/cm.yaml [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+kind: ConfigMap
+metadata:
+  name: ex01configmap
+  namespace: exercise01
+data:
+  MARIADB_HOST: "mariadb"
+  MARIADB_PORT_NUMBER: "3306"
+  MARIADB_USER: "bn_wordpress"
+  MARIADB_DATABASE: "bitnami_wordpress"
+
```
[the remark this answers is not in the document]
Comment (mangel99) [reply manuel_angel/session01/exercise01/cm.yaml]: ok

File: manuel_angel/session01/exercise01/cm.yaml [hunk 11]
```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ex01configmap
+  namespace: exercise01
+data:
```
[the remark this answers is not in the document]
Comment (mangel99) [reply manuel_angel/session01/exercise01/cm.yaml]: sure, but I didn't have time to make secrets, so I left it to exercise02...I know...I'm a mess

File: manuel_angel/session01/exercise01/deliver.bash [hunk 12]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (mangel99) [reply manuel_angel/session01/exercise01/deliver.bash]: I deleted the script, it was a sketch I made during development and I didn't have time to finish it, I shouldn't have uploaded it.
I'll try to finish it.

File: manuel_angel/session01/exercise01/deliver.bash [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+kubectl create -f cm.yaml
+## Create service
+kubectl create -f wordpress-svc.yaml
+kubectl create -f mariadb-svc.yaml
+
+## Create deployment
+kubectl create -f wordpress-deployment.yaml
+kubectl create -f wordpress-canary-deployment.yaml
+
+kubectl create -f mariadb-deployment.yaml
```
[the remark this answers is not in the document]
Comment (mangel99) [reply manuel_angel/session01/exercise01/deliver.bash]: agree, I manually launch them li this way, let's forget the script was only a sketch

File: manuel_angel/session01/exercise02/mariadb-slave-deployment.yaml (hunk 5 above)
Comment (mangel99) [reply manuel_angel/session01/exercise02/mariadb-slave-deployment.yaml]: I thought it didn't make much sense in the database, I removed it after I put it in.

File: manuel_angel/session01/exercise02/wordpress-svc.yaml (hunk 6 above)
Comment (mangel99) [reply manuel_angel/session01/exercise02/wordpress-svc.yaml]: ups, C&P

File: manuel_angel/session01/exercise02/wordpress-deployment.yaml (hunk 8 above)
Comment (mangel99) [reply manuel_angel/session01/exercise02/wordpress-deployment.yaml]: Of course, it's 02...I had fixed it but I would undo the changes before upload it .

**Pull request merged**